.github/workflows/ci-test-debian.yml

```diff
@@ -44,15 +44,17 @@ jobs:
 
   # Build with GCC and test P4C on Ubuntu 20.04.
   test-ubuntu20:
-    name: test-ubuntu20 (Unity ${{ matrix.unity }})
+    name: test-ubuntu20 (GTest ${{ matrix.gtest }}, Unity ${{ matrix.unity }})
     strategy:
       fail-fast: false
       matrix:
+        gtest: [ON, OFF]
```

Comment:
We unfortunately have limited CI resources, the more tests we add the slower they run. I would fold this into the UNITY=OFF test run, i.e, disable GTEST when we do not use a Unity build.

Reply:
Done.